<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeInfo</class>
 <widget class="QWidget" name="changeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>313</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>313</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>313</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>缺课日期</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="classnum_c">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>55</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>37</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>37</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>节课</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_c">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课程名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学生姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stuname_c">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入学生姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>缺课类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="method_c">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>55</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>迟到</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>旷课</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>早退</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>请假</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="classname_c">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入课程名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="right">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wrong">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID      </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="id_c">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>55</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="load">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>加载</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
